templates/home/index.html.twig

```diff
@@ -1,7 +1,58 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}Projektai{% endblock %}
+
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
-{% endblock %}
+
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for i in 0 .. teams|length - 1 %}
+            {% for student in students[i] %}
```

Comment:
Twig palaiko `for` sintaksę, kur einama per visus masyvo elementus. 

Kažkas panašaus į: 
```twig
 {% for student in students %}
```

Šiaip siūlyčiau plačiau pasiskaityti `Twig` funkcijų galimybes, nes su `for i in 0 .. teams|length - 1` nelabai kas programuoja

Plačiau:
 * https://twig.symfony.com/doc/2.x/tags/for.html#the-loop-variable